### ISSUE TITLE
left /right channel settings are not full preserved

### ISSUE THREAD

**Issue by Tirpitz93** [RTO-Chatter/ACE-TFAR-Setter#19]
When saving and loading channels with Left/right ear preferences these are not fully preserved when loaded.
both primary and alternate channel are put on the same ear.

**Comment by M3ales** [comment]
I'm guessing this is for ``TFAR Beta` specifically?

**Comment by Tirpitz93** [comment]
> I'm guessing this is for `TFAR Beta` specifically?

Yes, I can't say for the non beta as I don't use it, at least not regularly.
I will have a dig later some time
